aws-rds-customdbengineversion/inputs/inputs_1_create.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Engine": "custom-oracle-ee-cdb",
+  "DatabaseInstallationFilesS3BucketName": "{{S3BucketNamePrefix}}",
```

Comment:
Using resolution via `{{ResourceExportName}}` requires such resource to be already exported by a dependencies stack. Do we have such dependencies stack definition? If not, Contract Test run will not succeed.

Reply:
Dependencies stack is provided in deploying stage.